editor/types/tableMapConfig.ts

```diff
@@ -0,0 +1,57 @@
+import { FeatureCollection } from "geojson";
+import { LayerProps } from "react-map-gl";
+
+/**
+ * An index of functions that transform input data into a geojson feature collection
+ */
+export const geoJsonTransformers = {
```

Comment:
This can theoretically be expanded to handle other geometry types, like the location polygons. 

We can of course make a DB view that returns geojson directly, but by doing the transform client-side we're able to re-use the same query/query-builder that powers the table view.